### PR TITLE
autogluon_dashboard PyPI Package

### PR DESCRIPTION
**Pull request by shreyash2106**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -14,3 +14,50 @@ known_third_party = [
...
+
+[project]
+name = "autogluon_dashboard"
+version = "0.0.1"
+authors = [
+  { name="AutoGluon Community"},
+]
+description = "A dashboard package to view AutoGluon evaluation and benchmark metrics"
+readme = "README.md"
+requires-python = ">=3.9"
```
Comment (yinweisu) [inline pyproject.toml:30]: >=3.8, <3.10 to align with line 46-47 and autogluon-bench

File: src/autogluon_dashboard/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import subprocess
+import sys
+
+from autogluon.common.loaders import load_pd
+
+if __name__ == "__main__":
+    csv1 = sys.argv[2]
+    csv2 = sys.argv[4]
+    # subprocess.run(["panel", "serve", "src/autogluon_dashboard/app.py", "--autoreload", "--args", f"{csv1}", f"{csv2}"])
```
Comment (yinweisu) [inline src/autogluon_dashboard/dashboard.py:9]: address in a follow-up PR, but ideally we should take in args from the user regarding whether we want to `serve` or `convert`

File: pyproject.toml (hunk 1 above)
Comment (shreyash2106) [reply pyproject.toml:30]: Done!

File: src/autogluon_dashboard/dashboard.py (hunk 2 above)
Comment (shreyash2106) [reply src/autogluon_dashboard/dashboard.py:9]: Sure!

**Review by yinweisu — APPROVED** [review]
LGTM

**Pull request merged**